app/src/components/Home/CoverPage.jsx

```diff
@@ -122,6 +131,13 @@ class CoverPage extends Component {
       beforeChange: (currentSlider, nextSlider) => {
         this.onSliderChange(nextSlider);
       },
+      afterChange: this.state.windowWidth <= 768 ? () => {
```

Comment:
Two comments here. First, I would prefer to have the condition inside the callback than having a tertiary operator returning or a function or `null`. Second, I think you're way of determining if "Leo"' slide is active is wrong: the `afterChange` callback receives the current active slide. I would thus save in a variable the index of the slide and just check if the current one matches it.